tools/solidity-coverage-example/README.md

```diff
@@ -0,0 +1,34 @@
+# Sample Hardhat Project
+
+This project demonstrates a basic Hardhat use case with Hedera. It comes with a sample contract, a test for that contract, and a script that deploys that contract.
+
+Try running some of the following tasks:
+
+```shell
+npx hardhat help
+npx hardhat test
+REPORT_GAS=true npx hardhat test
```

Comment:
Q: what's this for?

Reply:
It's part of the default **README.md** generated by the hardhat initial setup. I'll remove it.